Kickstarter-iOS/Views/Controllers/LiveStreamContainerViewController.swift

```diff
@@ -110,7 +110,7 @@ public final class LiveStreamContainerViewController: UIViewController {
     super.bindStyles()
 
     _ = self
-      |> baseControllerStyle()
+      |> baseControllerFeedStyle()
```

Comment:
i think this should also go back to `baseControllerStyle` cause it's just a plain view controller and it sets its own background color